google-cloud-spanner-executor/src/main/java/com/google/cloud/executor/spanner/CloudExecutorImpl.java

```diff
@@ -48,6 +57,17 @@ public StreamObserver<SpannerAsyncActionRequest> executeActionAsync(
       @Override
       public void onNext(SpannerAsyncActionRequest request) {
         LOGGER.log(Level.INFO, String.format("Receiving request: \n%s", request));
+
+        // Use Multiplexed sessions for all supported operations if the
+        // multiplexedSessionOperationsRatio from command line is > 0.0.
+        if (multiplexedSessionOperationsRatio > 0.0) {
+          SessionPoolOptions.Builder sessionPoolOptions =
+              SessionPoolOptions.newBuilder().setUseMultiplexed(true);
```

Comment:
Instead of `newBuilder` use `toBuilder`. That way if something is already set in the action input then it won't get over-written.